// databases/src/main.js
var mysql = require('mysql')
var mongodb = require('mongodb')

var DB = {
	customers: function(){
		return mysql.createPool({
			connectionLimit : 100, //important
			host : "localhost",
			user : "gasmon",
			password : "OniGas2016!!",
			database : "Customers",
			debug : false,
		});
	},
	sensors: function(){
		return mysql.createPool({
			connectionLimit : 100, //important
			host : "localhost",
			user : "gasmon",
			password : "OniGas2016!!",
			database : "Sensors",
			debug : false
		});
	},
	messages: function(){
		mysql.createPool({
			connectionLimit : 100, //important
			host : "localhost",
			user : "gasmon",
			password : "OniGas2016!!",
			database : "Messages",
			debug : false,
		});
	},
	companies: function(){
		mysql.createPool({
			connectionLimit : 100, //important
			host : "localhost",
			user : "gasmon",
			password : "OniGas2016!!",
			database : "Companies",
			debug : false,
		});
	},
	purchases: function(){
		mysql.createPool({
			connectionLimit : 100, //important
			host : "localhost",
			user : "gasmon",
			password : "OniGas2016!!",
			database : "Purchases",
			debug : false,
		});
	}
}

module.exports = DB;
